src/utils.rs

```diff
@@ -125,6 +125,24 @@ where
     }))
 }
 
+pub fn de_without_slashes<'de, D>(deserializer: D) -> Result<String, D::Error>
+where
+    D: ::serde::Deserializer<'de>,
+{
+    use serde::Deserialize;
+    let s = String::deserialize(deserializer)?;
+    Ok(s.replace("/", ""))
+}
+
+pub fn de_option_without_slashes<'de, D>(de: D) -> Result<Option<String>, D::Error>
+where
+    D: ::serde::Deserializer<'de>,
+{
+    use serde::Deserialize;
+    let option = Option::<String>::deserialize(de)?;
+    Ok(option.and_then(|s| Some(s.replace("/", ""))))
```

Comment:
```rust
Ok(option.map(|s| s.replace("/", "")))
```
I think you can use `de_without_slashes` in `de_option_without_slashes` or the other way round